.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,20 @@
+# Issue Description
+  [Type of Issue]
+
+## Problem
+  [Describe what the problem we are trying yo solve.]
+
+## Success
+  [What this task should accomplish when it's considered completed.]
+
+### Requirements
+
+- [ ] Add
+- [ ] Requirements
+- [ ] here
+
+## Implementation notes
+ [Suggest how this might be implmented.]
+
+
+### Screenshot (If Applicable)
```

Comment:
Is this to show the bug if it is trying to change it or the future feature design? What is this the screenshot of?

Reply:
If applicable, consider including screenshots to provide a clearer visual representation of the issue (e.g., bug, feature request) to enhance understanding and easy code review.

Reply:
@karlk08 Screenshot or video for context.

#Maybe something like that if applicable